indi-asi/asi_ccd.cpp

```diff
@@ -1023,6 +1036,51 @@ bool ASICCD::StartExposure(float duration)
 {
     ASI_ERROR_CODE errCode = ASI_SUCCESS;
 
+    long blinks = BlinkN[0].value;
+    if (blinks > 0)
+    {
+        LOGF_INFO("Blinking %ld time(s) before exposure", blinks);
+
+        const long duration = 0;
+        errCode = ASISetControlValue(m_camInfo->CameraID, ASI_EXPOSURE, duration, ASI_FALSE);
```

Comment:
This is using duration which is the exposure time? isn't this suppose to be short 100ms or something?

Reply:
Yes, it is exposure time. 0 seems to work for me. I have zero experience with ASI and CMOS idiosyncrasies... so I can't really prove it is the best one. Probably, I can make it configurable too.

Reply:
...done.